bluebrain/repo-bluebrain/packages/py-cgal-pybind/package.py

```diff
@@ -7,10 +7,10 @@
 
 
 class PyCgalPybind(PythonPackage):
-    """Internal Python bindings for CGAL"""
+    """Python bindings for CGAL"""
 
-    homepage = "https://bbpgitlab.epfl.ch/nse/cgal-pybind"
-    git = "git@bbpgitlab.epfl.ch:nse/cgal-pybind.git"
+    homepage = "https://github.com/BlueBrain/cgal-pybind"
+    git = "https://github.com/BlueBrain/cgal-pybind.git"
 
     version("develop", submodules=True)
     version("0.1.3", tag="cgal-pybind-v0.1.3", submodules=True)
```

Comment:
On github only 0.1.4 is tagged, maybe we can add it and keep only this one?
If it's ok to use this version, we can also use pypi instead of tags, since that version is available on https://pypi.org/project/cgal-pybind